lib/find-file.js

```diff
@@ -53,9 +53,13 @@ function findFile(opt) {
       // find path to the root, stopping at cwd, package.json or bower.json
       var pathToRoot = [];
       var isWorking;
+      var depthCount = 0;
       do {
         pathToRoot.push(absoluteStart);
-        isWorking = testWithinLimit(absoluteStart);
+        if (!testNotPackage(absoluteStart)) {
+          depthCount += 1;
+        }
+        isWorking = testWithinLimit(absoluteStart) && depthCount < options.moduleDepth;
```

Comment:
**An observation:** Code searches upward/shallower from the original path. The code that does deeper is below  `.filter(testNotPackage)`. Which one of these solves your use case, shallower or deeper?